### PR TITLE
Use StorageClient to talk to storage service.

### PR DESCRIPTION
**Pull request by mbarnes**
Just some code simplification to make storage service calls less tedious.

Tests required extensive updates.

Requires: https://github.com/projectatomic/commissaire/pull/63

Re-run tests after the `commissaire` PR is merged.

### CODE REVIEW

**Comment by mbarnes** [comment]
Hang on... still got a bit more work to do on this.

**Comment by ashcrow** [comment]
bot, retest this please

File: src/commissaire_http/handlers/clusters/__init__.py [hunk 1]
```diff
@@ -268,13 +256,10 @@ def update_cluster_members(message, bus):
...
-    cluster = models.Cluster.new(**cluster['result'])
-    cluster._validate()
-    response = bus.request(
-        'storage.save', params=[
-            'Cluster', cluster.to_dict()])
-    return create_response(message['id'], response['result'])
+    cluster.hostset = list(new_hosts)
+    saved_cluster = bus.storage.save(cluster)
+    # XXX Using to_dict() instead of to_dict_safe() to include hostset.
+    return create_response(message['id'], saved_cluster.to_dict())
```
Comment (ashcrow) [inline src/commissaire_http/handlers/clusters/__init__.py]: :+1:

**Comment by mbarnes** [comment]
[(3) [multi-host-mgr-2] Decrease service complexity](https://trello.com/c/Gzjkpge2/561-3-multi-host-mgr-2-decrease-service-complexity)

**Pull request merged**